nodeup/pkg/model/kubecontrollermanager.go

```diff
@@ -188,6 +189,14 @@ func (b *KubeControllerManagerBuilder) buildPod() (*v1.Pod, error) {
 		},
 	}
 
+	if os.Getenv("HTTP_PROXY") != "" || os.Getenv("HTTPS_PROXY") != "" {
```

Comment:
@johnzeringue putting the helper functions into upup is probably the best place for the code to live.  There is not an official convention.  If you want to recommend a convention let us know.